articles/00-GettingStarted.md

```diff
@@ -66,32 +84,94 @@ Which contains the following:
+> Will will be updating to support enabling XR through the new system in a future release.
+
+Once you are ready to build, open the Unity Build settings, and switch to the target platform you wish to build on.
 
 > **Note:** Depending on the platform's simulation support, you should be able to run in the editor by pressing play.
 
-Once satasfied, you can build your project by using the Mixed Reality Toolkit's Build window:
+For specific builds, please check the quickstart sections below:
+
+> Please check the FAQ for any difficulties.
+
+### Building for Windows Standalone (OpenVR/Oculus Rift-Link)
+
+When targeting Windows Standalone there should be no special setup and the Mixed Reality Toolkit will work as configured. Builds should be able to target OpenVR and Oculus Quest (via Oculus Link).
+
+Please ensure either the OpenVR or Oculus Camera and Controller providers are configured.
+
+### Building for Windows Mixed Reality (UWP)
+
+Unity allows running of UWP projects directly from the editor for simple testing.  Builds that are exported will require Visual Studio with the latest UWP API's installed to produce a build to ship and run on other WIndows 10 machines or the HoloLens.  Visual Studio also provides an independent method to publish to the Windows Store for distribution.
+
+Please ensure the `Windows Mixed Reality` XRTK package is installed and that the Windows Mixed Reality Camera and controller providers are configured.
+
+### Building for Oculus Quest / Android
+
+Building for Oculus Quest is rather unique, if you have your Oculus Quest connected to your machine via a high speed USB3 cable and have the Oculus Client running, you can run Quest project in Windows Standalone mode directly on the Quest via Oculus Link, [please see Oculus Support for further details](https://support.oculus.com/444256562873335/).  This will enable both controller and hands support for quick development.
+
+When building for Android, there is an additional requirement to install an `AndroidManaifest` for the quest to operate correctly, the XRTK provides a fast an easy way to do this by simply clicking `Mixed Reality Toolkit -> Tools -> Oculus -> Create Oculus Quest Compatible AndroidManaifest.xml` in the Unity editor menu. (**without this, your project may not start or appear in 2D**)
+
+Please ensure the `Oculus` XRTK package is installed and that the Oculus Camera and controller providers are configured.
+
+### Building for Magic Leap
+
+Magic Leap is the one exception that does require the new Unity XR Management system and will require a separate Unity project at this time, primarily due the way the Magic leap platform is evolving.  We recommend building your project for one of the other platforms and then simply copying the asset folder to a new Unity project for Magic Leap to build and run.
```

Comment:
>  will require a separate Unity project at this time

What does this even mean? I'm not sure where this information came from.

>  We recommend building your project for one of the other platforms and then simply copying the asset folder to a new Unity project for Magic Leap to build and run.

Why?

Reply:
You cannot have the legacy XR system AND the new XR Plugin Management system enabled at the same time. 
Unless you are saying that Magic leap still works with the legacy XR system only?

Reply:
Depends on which XR Plugin version you install